torchci/test/fixtures/pull_request_reviews.json

```diff
@@ -0,0 +1,119 @@
+[
```

Comment:
Might be good to add this as a generated file to .gitattributes so these don't show up in PR review

Reply:
Can these be autogenerated? I copy/pasta-ed this file from a github api